Lib/tokenize.py

```diff
@@ -183,6 +184,23 @@ def add_whitespace(self, start):
         if col_offset:
             self.tokens.append(" " * col_offset)
 
+    def escape_brackets(self, token):
+        characters = []
+        consume_until_next_bracket = False
+        for character in token:
+            if character == "}":
+                if consume_until_next_bracket:
+                    consume_until_next_bracket = False
+                else:
+                    characters.append(character)
+            if character == "{":
+                if characters[-2:] != ["\\", "N"]:
```

Comment:
slash escapement missing? i.e: "f'\\\\N{{x}}'"

Reply:
Fixed in [4ab4352](https://github.com/python/cpython/pull/115171/commits/4ab43521a435827f2227e00bc9e160a8d3149154)